### ISSUE TITLE
performance-standards and cdn-caching-strategy miss caching that core-web-vitals' own analyzer confirms

### ISSUE THREAD

**Issue by ruleofcodedev** [Rule-Of-Code/RoC#103]
**Version:** 7.21.0
**Laws:** `performance-standards`, `cdn-caching-strategy` — against
`core-web-vitals-compliance`

Same family as #95, which you closed: a concern that is met, reported as absent
because it is expressed in the framework's idiom rather than webpack's. Two more
instances, one of them a direct disagreement inside a single run.

## 1. Two caching analyzers, opposite verdicts, same audit

```
performance-standards      → "Browser caching strategy not implemented"
core-web-vitals-compliance → does not report it
```

`core-web-vitals`'s analyzer accepts three strategies and finds ours. Asked
directly:

```js
BrowserCachingAnalyzerService.analyze(projectRoot)
// { hasStrategy: true, strategies: ['HTTP caching headers configured'] }
```

It reads `firebase.json` and checks for `Cache-Control` / `ETag` / `Expires`
(`browser-caching.analyzer.js:59`). Ours declares them:

```jsonc
// firebase.json
{ "source": "**/*.@(js|css|woff2)",
  "headers": [{ "key": "Cache-Control", "value": "public, max-age=31536000, immutable" }] },
{ "source": "**/*.@(ico|svg|png|webp|jpg|jpeg|avif)",
  "headers": [{ "key": "Cache-Control", "value": "public, max-age=86400" }] }
```

`performance-standards` has its own analyzer
(`performance-standards/services/caching-strategy.analyzer.js`) which returns
clean only for a **service worker** or a Capacitor app, and never reads
`firebase.json`:

```js
if (this.hasServiceWorkerConfigInFiles(configFiles)) return violations;
violations.push('Browser caching strategy not implemented');
```

So a prerendered static site with correct immutable caching is told it has no
caching strategy, by one law, while another law in the same run has already
confirmed it does.

Its remediation — "Implement service worker and HTTP caching headers" — is half
done and half wrong for this shape: a fully prerendered site served from a CDN
gains nothing from a service worker except a second cache to invalidate.

## 2. Asset hashing is looked for in webpack's vocabulary

```
cdn-caching-strategy → "Static asset optimization not configured"
```

```js
OPTIMIZATION_KEYWORDS = ['contenthash', 'chunkhash', 'splitChunks'];
```

Those are webpack strings. Angular's builder declares the same thing as:

```jsonc
// apps/web/project.json
"configurations": { "production": { "outputHashing": "all" } }
```

Every asset we ship is content-hashed — `IBMPlexSerif-<hash>.woff2`,
`main-<hash>.js` — which is why the immutable header above is safe, and why the
font preloads have to be stamped in after the build (the case in #95). The word
`contenthash` appears nowhere in an Angular project, so this finding is
unreachable for the whole ecosystem, not just for us.

## Why not simply waive them

Our policy is that a law is `error` or it is declared not-applicable with a
written reason. Neither fits: the concern exists, we meet it, and the checker
cannot see it — so both sit at `warning`, which is the state we are trying not
to have.

## Suggested direction

- **Caching**: let `performance-standards` use the same analyzer
  `core-web-vitals-compliance` already has. It handles the three real strategies
  and it is right about ours today. One reader, one verdict — the shape 7.20.0
  applied to the branch-name laws.
- **Asset hashing**: add `outputHashing` to `OPTIMIZATION_KEYWORDS`, or read the
  build target's configuration when `project.json` / `angular.json` is present.
  `nx.json` beside it is a reliable marker.

Worth checking whether the same webpack vocabulary appears in other
performance laws — `splitChunks` has no Angular equivalent either, and a search
for it will find none of an Angular app's real chunking.
